backend/tracim_backend/templates/mail/upload_permission_to_receiver_body_html.mak

```diff
@@ -0,0 +1,17 @@
+<p>${_('Hello,')}</p>
+<p>${_('{username} allowed you to upload on sharespace {workspace_name}').format(username=emitter.display_name, workspace_name=workspace.label)}</p>
```

Comment:
Wrong name `sharespace`, its `shared space`.

Reply:
Do we need to give '{workspace_name}' ? Its not possible to just give workspace id for exemple?

Reply:
in fact we should not give workspace name anymore. but it's still unclear how we give information about workspace id/ upload_permission id.

I do think, as this look a bit technical, we should maybe not show too much. just add a small line at the bottom meaning this is upload permission XXX relating to workspace XXX may be just enough a troubleless. But i may be wrong.

Reply:
@buxx what do you think about that?

Reply:
fixed by refactoring of email template